validator/src/main.rs

```diff
@@ -44,7 +44,8 @@ use {
         runtime_config::RuntimeConfig,
         snapshot_config::{SnapshotConfig, SnapshotUsage},
         snapshot_utils::{
-            self, create_all_accounts_run_and_snapshot_dirs, ArchiveFormat, SnapshotVersion,
+            self, create_all_accounts_run_and_snapshot_dirs, create_and_canonicalize_directories,
```

Comment:
While we're at it, might as well use the newly created helper to de-duplicate the below instance as well:
https://github.com/solana-labs/solana/blob/bfe171017c3b15fd8b591b7f3773ab05a866f8a2/validator/src/main.rs#L965-L970

Reply:
done: 04e8f94b7f